baseplate/frameworks/queue_consumer/kombu.py

```diff
@@ -81,6 +88,44 @@ def stop(self) -> None:
         self.should_stop = True
 
 
+class KombuBatchConsumerWorker(ConsumerMixin, PumpWorker):
+    def __init__(
+        self,
+        connection: kombu.Connection,
+        queues: Sequence[kombu.Queue],
+        work_queue: BatchedQueue,
+        serializer: Optional[KombuSerializer] = None,
+    ) -> None:
+        self.connection = connection
+        self.queues = queues
+        self.work_queue = work_queue
+        self.serializer = serializer
+
+    def get_consumers(self, Consumer: kombu.Consumer, channel: Channel) -> Sequence[kombu.Consumer]:
```

Comment:
:no_bell: i know this is a kombu-ism, but the capitalized parameters really freak me out